addons/block_code/simple_nodes/simple_character/simple_character.gd

```diff
@@ -31,6 +31,36 @@ const PLAYER_KEYS = {
 	}
 }
 
+const PLAYER_JOYSTICK_BUTTONS = {
+	"up": JOY_BUTTON_DPAD_UP,
+	"down": JOY_BUTTON_DPAD_DOWN,
+	"left": JOY_BUTTON_DPAD_LEFT,
+	"right": JOY_BUTTON_DPAD_RIGHT,
+}
+
+const PLAYER_JOYSTICK_MOTION = {
+	"up":
+	{
+		"axis": JOY_AXIS_LEFT_Y,
+		"axis_value": 1,
+	},
+	"down":
+	{
+		"axis": JOY_AXIS_LEFT_Y,
+		"axis_value": -1,
+	},
```

Comment:
These are upside-down: pushing the joystick up moves the character down.

Reply:
fixed in commit dafe38e